e2e/scaffolding/extrinsicHelpers.ts

```diff
@@ -182,7 +182,7 @@ export class Extrinsic<N = unknown, T extends ISubmittableResult = ISubmittableR
   }
 
   // This uses automatic nonce management by default.
-  public async signAndSend(inputNonce?: AutoNonce, options: Partial<SignerOptions> = {}) {
+  public async signAndSend(inputNonce?: AutoNonce, options: Partial<SignerOptions> = {}, waitForInBlock = true) {
```

Comment:
`waitForInBlock` instead of `waitForFinalization` so that the boolean logic below is simple